src/main/groovy/io/seqera/wave/controller/RegistryProxyController.groovy

```diff
@@ -199,8 +204,12 @@ class RegistryProxyController {
         log.info "> Request [$httpRequest.method] $httpRequest.path"
         final route = routeHelper.parse("/v2/" + url)
 
+        String ip = addressResolver.resolve(httpRequest)
+        //fire a pull event
+        if( httpRequest.method == HttpMethod.GET )
```

Comment:
This count https requests, not pull. Use events should be triggered by 
* [increaseWavePullsCounter](https://github.com/seqeralabs/wave/blob/1abce25c0b8c066b0f6ee5183bf82c5116d75bdc/src/main/groovy/io/seqera/wave/controller/RegistryProxyController.groovy#L162-L162)
* [increaseFusionPullsCounteer](https://github.com/seqeralabs/wave/blob/1abce25c0b8c066b0f6ee5183bf82c5116d75bdc/src/main/groovy/io/seqera/wave/controller/RegistryProxyController.groovy#L185-L195) 
* [incrementBuildCounter](https://github.com/seqeralabs/wave/blob/1288538fe3c12e7bca81333b8da5ab8b5a4a494e/src/main/groovy/io/seqera/wave/service/builder/ContainerBuildServiceImpl.groovy#L225-L225)

Currrent logic using the metrics server, can be discarded